src/olympia/versions/admin.py

```diff
@@ -48,6 +50,27 @@ class Media:
     )
     inlines = (VersionReviewerFlagsInline,)
 
+    def addon_guid(self, obj):
+        return obj.addon.guid
 
+    addon_guid.short_description = 'Add-on GUID'
+
+
+class InstallOriginAdmin(admin.ModelAdmin):
```

Comment:
what's the use case for this admin?  Admins can't add, delete, or change the records, and the table will be a long list of ids next to origins and domains (without filtering), which appears of limited use.

Reply:
is it only for QA?

Reply:
Mostly for QA/debugging at the moment, yeah. There hasn't been anything specced for admin needs yet.